src/librustdoc/lib.rs

```diff
@@ -162,8 +162,9 @@ pub fn opts() -> Vec<RustcOptGroup> {
                       "URL to send code snippets to", "URL")),
         stable(optflag("", "markdown-no-toc", "don't include table of contents")),
         stable(optopt("e", "extend-css",
-                      "to redefine some css rules with a given file to generate doc with your \
-                       own theme", "PATH")),
+                      "To redefine some css rules with a given file to generate doc with your \
```

Comment:
s/redefine/add
s/css/CSS

I'm not sure about the "add", but I think "redefine" sends the wrong message. You are not changing existing CSS rules, just adding more rules, that may be more specific (because of [specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)) and thus appear like they replaced the default rules.

In addition to that it is also totally possible to add new rules for that `.fancy-table` you used in your docs.

(Sorry to continue do nitpick on this)